### PR TITLE
NAS-109825 / 21.06 / Enabled custom empty config messages & fixed messages for SMB table

### PR DESCRIPTION
**Pull request by ryousaf**
NAS-109825

### CODE REVIEW

File: src/app/pages/common/entity/entity-table/entity-table.component.ts [hunk 1]
```diff
@@ -418,18 +426,105 @@ export class EntityTableComponent implements OnInit, AfterViewInit, OnDestroy {
...
+      this.configureEmptyTable(this.dataSource.filter ? EmptyType.no_search_results : this.firstUse ? EmptyType.first_use : EmptyType.no_page_data);
     }
 
     if (this.dataSource.paginator && this.conf.config.paging) {
       this.dataSource.paginator.firstPage();
     }
   }
+
+  configureEmptyTable(emptyType: EmptyType, error: any = null) {
+    if(emptyType && emptyType === EmptyType.loading) {
```
Comment (denysbutenko) [inline src/app/pages/common/entity/entity-table/entity-table.component.ts:438]: This function takes time to understand the logic. Wouldn't it be better to use the "switch" statement on enum for better readability?

Comment (ryousaf) [reply src/app/pages/common/entity/entity-table/entity-table.component.ts:438]: Good idea. I'll add that in.

Comment (denysbutenko) [reply src/app/pages/common/entity/entity-table/entity-table.component.ts]: Thanks!

**Review by AlexKarpov98 — APPROVED** [review]
LGTM

**Review by denysbutenko — APPROVED** [review]

**Pull request merged**